ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -25,6 +25,14 @@ CH_SS_SWTC:
   - SS
   - SWTC
 
+CH_SS_SCHWTC:
+  description: " Simple air cooled chiller."
+  is_canonical: true
+  implements:
+  - CH
+  - SS
+  - SCHWTC
```

Comment:
Add the field you need (`cooling_percentage_command`) as an optional field on `SWTC` and reuse existing type `CH_SS_SWTC`.